homeassistant/components/dynalite/const.py

```diff
@@ -1,7 +1,7 @@
 """Constants for the Dynalite component."""
 import logging
 
-from homeassistant.const import CONF_ROOM
+from homeassistant.const import CONF_DEFAULT, CONF_ROOM  # NOQA
```

Comment:
Please address this pylint warning, thanks! 👍

Reply:
will take a look

Reply:
if CONF_DEFAULT is imported here in the individual files, why do we want it imported in const.py as well?

Reply:
IMHO this occurance needs to be removed, that should cover it.

Reply:
i agree. the one i mentioned in line 22 was already removed and replaced by this. Only import CONF_ROOM.
The second thing that comes to mind is why not do the same for CONF_ROOM as for CONF_DEFAULT? Is there a specific reason one was moved from dynalite.const and the other wasn't? not an experienced python programmer, so not sure whether this is intentional

Reply:
It is best to use core constants whenever possible. The only issue with not importing CONF_DEFAULT into .const is with tests. They fail without it

Reply:
Found the solution. dynalite.CONF_DEFAULT = CONF_DEFAULT